dkpro-core-lancaster-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/lancaster/LancasterStemmer.java

```diff
@@ -0,0 +1,144 @@
+package de.tudarmstadt.ukp.dkpro.core.lancaster;
```

Comment:
https://github.com/dkpro/dkpro-core/blob/master/dkpro-core-snowball-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/snowball/SnowballStemmer.java has a header. Are you referring to a different SnowballStemmer?